docs/groqd-playground.mdx

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_position: 7
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Groqd Playground (Plugin)
+
+Groqd Playground is a plugin for Sanity Studio for testing groqd queries, featuring:
+
+- a TypeScript editor experience with syntax/type highlighting;
+- parsed response and error messages for when responses fail to pass Zod validation;
+- dataset/api version switchers.
+
+![Sample of groqd playground in use](./img/groqd-playground-sample.png)
+
+## Installation
+
+In your Sanity Studio project, install `groqd-playground` using your favorite package registry tool.
+
+<Tabs>
+<TabItem value="npm" label="NPM">
+
+```sh
+npm install groqd-playground
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```sh
+yarn add groqd-playground
+```
+
+</TabItem>
+<TabItem value="pnpm" label="pnpm">
+
+```sh
+pnpm add groqd-playground
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+**The playground tool has the following peer dependencies**, most/all of which you should already have installed in a Sanity project:
```

Comment:
I had to read this a couple times. I think we can say it has these peer does, which are typically already a part of most Sanity Studio implementations: icons, etc...